presto-main/src/test/java/io/prestosql/security/TestFileBasedSystemAccessControl.java

```diff
@@ -124,10 +123,16 @@ public void testDocsExample()
 
         accessControlManager.checkCanImpersonateUser(Identity.ofUser("alice"), "charlie");
         accessControlManager.checkCanImpersonateUser(Identity.ofUser("bob"), "charlie");
-        assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("alice"), "bob"));
-        assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("bob"), "alice"));
-
-        assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("charlie"), "doris"));
+        assertThatThrownBy(() -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("alice"), "bob"))
+                .isInstanceOf(AccessDeniedException.class)
+                .hasMessageContaining("Access Denied: User alice cannot impersonate user bob");
+        assertThatThrownBy(() -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("bob"), "alice"))
+                .isInstanceOf(AccessDeniedException.class)
+                .hasMessageContaining("Access Denied: User bob cannot impersonate user alice");
+
+        assertThatThrownBy(() -> accessControlManager.checkCanImpersonateUser(Identity.ofUser("charlie"), "doris"))
+                .isInstanceOf(AccessDeniedException.class)
+                .hasMessageContaining("Access Denied: User charlie cannot impersonate user doris");
```

Comment:
The quality of access control messages is very important from end user perspective.
Thanks for doing this. This is a great testing improvement especially here.